types/mempool/priority_nonce.go

```diff
@@ -92,6 +93,14 @@ func PriorityNonceWithOnRead(onRead func(tx sdk.Tx)) PriorityNonceMempoolOption
 	}
 }
 
+// PriorityNonceWithTxReplacement sets a callback to be called when deplicated tx nonce detected
+// during the mempool insert, the dev can define the tx replacement rule based on the tx priority
+func PriorityNonceWithTxReplacement(txReplacementRule func(oldTxPriority, newTxPriority int64) bool) PriorityNonceMempoolOption {
```

Comment:
Could you add a quick mention of this option here: https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-apps/02-app-mempool.md#L138-L139

Reply:
updated